code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -74,6 +87,7 @@
 	if(!candidates.len || !owner)
 		bursting = FALSE
 		stage = 4
+		advance_embryo_stage()	//We stop growing at stage 5, so if we go back a stage, we need to be able to advance again.
```

Comment:
Won't this immediately poll the ghosts again? Since, the stage is 4, is immediately ++'d to 5.

Shouldn't this be instead a 60 timer to calling advance_embryo_stage?

Reply:
Good catch!
Thank you, it would do this. The proc was arranged differently when I made this change, and I evidently forgot to change this to reflect the proc's changes.